### PR TITLE
New clean docs

### PR DESCRIPTION
**Pull request by Hackdom**
Finally found the setting that was giving whitespace issues. This should do.

### CODE REVIEW

File: Wallet.sol [hunk 1]
```diff
@@ -400,6 +432,18 @@ contract Wallet is multisig, multiowned, daylimit, creator {
 
 	// INTERNAL METHODS
 
+	/// @dev Creates new contract
+	/// @param _value Amount to send
+	/// @param _code Code for new contract
+	/// @return o_addr Address of new contract
+	function create(uint _value, bytes _code) internal returns (address o_addr) {
+		assembly {
```
Comment (5chdn) [inline Wallet.sol]: Again this is not docs-only.

Comment (Hackdom) [reply Wallet.sol]: hmmm... hold on

Comment (Hackdom) [reply Wallet.sol]: Did this come from the master branch? I didn't make that change explicitly, I think it came from pulling master and then merging into the whg branch

Comment (Hackdom) [reply Wallet.sol]: maybe not, master has it the other way also, it pulled from somewhere and didn't get that commit...

**Comment by keorn** [comment]
This is into whg branch, is it meant to go into master?

**Comment by Hackdom** [comment]
Eventually, we were just pushing into that branch in the summer, wasn't sure if that changed.

**Comment by keorn** [comment]
That branch is unused, please rebase on master.

**Pull request closed without merging**